src/sdk/trace/span_processor.rs

```diff
@@ -269,6 +283,56 @@ impl BatchSpanProcessor {
             config: Default::default(),
         }
     }
+
+    /// Create a new batch processor builder and set the config value based on environment variables.
+    pub fn from_env<E, S, SO, I, IO>(
```

Comment:
Do you prefer the explicit `from_env` here? Has advantages of being clear when it is looking from env, alternative could be to have the batch config be options and fall back to checking env variables for any unset values sorta merging both constructors?

Reply:
I guess that depends on whether we should assume users always want to apply environment variable configuration. Personally I'd like know exactly where does the configuration comes from. But we could use environment variable to fill unset values in configs too.

Reply:
Yeah I see the pros and cons of both (e.g. the jaeger exporter has an explicit `from_env`). I think it is fine for now to get feedback from the community on which is preferred.